audit/projects/k8s-conform/buckets/k8s-conform-huaweicloud/iam.json

```diff
@@ -15,6 +15,12 @@
       ],
       "role": "roles/storage.legacyBucketReader"
```

Comment:
Reader seems wrong on these three, and is left over from the same oldr bug.  Will remove.

Reply:
I removed that manually - if you run again you should find these 3 changes